File: aug_22_tabular/jl/baseline.jl
using CSV
using DataFrames
using Statistics

trn = CSV.read("./aug_22_tabular/data/train.csv", DataFrame)
tst_id = CSV.read("./aug_22_tabular/data/test.csv", DataFrame).id

pred = mean(trn.failure)

sub = DataFrame(
    id = tst_id,
    failure = pred
)

File: aug_22_tabular/R/baseline.R
library(tidyverse)

trn <- read_csv(here::here("aug_22_tabular/data/train.csv"))

tst_id <- read_csv(here::here("aug_22_tabular/data/test.csv"))$id

pred <- mean(trn$failure)

sub <- tibble(
    id = tst_id,
    failure = pred
)

write_csv(sub, here::here("aug_22_tabular/submissions/baseline_r.csv"))
